components/lib/calendar/Calendar.vue

```diff
@@ -1170,6 +1170,7 @@ export default {
                     this.pm ? date.setHours(this.currentHour + 12) : date.setHours(this.currentHour);
                 }
 
+                date.setHours(this.currentHour);
```

Comment:
Correct code would be 
`else date.setHours(this.currentHour);`

Reply:
oh this issue is resolved the PR (https://github.com/primefaces/primevue/pull/5607)